lagunita/lms/static/sass/partials/base/custom_overrides/_course_styling.scss

```diff
@@ -1,3 +1,13 @@
+#course-container {
+    .welcome-message {
+        .dismiss-message {
+            width: 100%;
+            text-align: right;
+            padding-bottom: 10px;
+        }
+    }
+}
+
```

Comment:
Does this file get shared by the other themes?

If not, should we port this over to `suclass` too?

Reply:
Otherwise, the screenshots look good.

Reply:
@stvstnfrd ^

Reply:
I mean, we want these changes for other instances too, right?

Reply:
Should I do this for CME as well, or just SUclass for now?

Reply:
I honestly don't know why we wouldn't use these styles for every single theme..

Reply:
Mostly because we're not deploying ginkgo to CME...